### ISSUE TITLE
2.4.0 Startup Crash

### ISSUE THREAD

**Issue by 04kaia** [FTBTeam/FTB-Revelation-2#96]
* **Modpack Version**: 2.4.0 Beta

* **Issue**: Game Crashing on Launch

* **Link to Log or Crash File Paste**: https://paste.feed-the-beast.com/view/532cecff  https://paste.feed-the-beast.com/view/0c6240cf
* **Is it Repeatable?**: Yes. Launch game with base mod pack installed, remove Gendustry to repeat Extra Utils crash.

* **Mod/s Affected**: Gendustry is listed as the cause, once removed Extra utilities 2 is listed as the cause.

* **Known Fix**: Remove both Gendustry and Extra Utilities 2

**Comment by iMagic16** [comment]
+1 exact same issue, once both are removed it'll boot fine

**Comment by iMagic16** [comment]
also crashing on 2.3.0 now - Windows Update happened overnight (17314 in dxdiag, 1803 in "About"). Could be that!

**Comment by Mauvai** [comment]
I have the same issue, but the mod after gendustry is thermal expansion...

**Comment by railstar0083** [comment]
I am on Windows 7 still and I am having the same issue.  Gendustry fails, then if you remove it, extrautils fails.

**Comment by Flockhammer** [comment]
same crash and I am just now downloading the update for windows. 

The game crashed whilst there was a severe problem during mod loading that has caused the game to fail
Error: net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from GenDustry (gendustry)

**Comment by megadeth116** [comment]
Same crash for me. Tried to go back one version (played yesterday) and still crashes even when deleted and installed again (on the 2.3 update). No updates of Java nor windows (using win 7).

**Comment by Flockhammer** [comment]
I was able to get it to load by changing to the JAR launcher with the Beta version. I'm downloading the stable release atm and will see if that loads as well.

**Comment by Flockhammer** [comment]
So if you are having issues try changing to loading with the Jar Launcher over the Native launcher. Its working for me atm.

**Comment by shadoxxhd** [comment]
fix: https://drive.google.com/drive/folders/1iLvRGb9leb9SZ8CR_r0KYPjhHFEQnFjR
The problem was a typo in language files published my minecraft devs, causing a NPE in some mods.
Just replace your mod jars with the ones provided above (class mods also available there).

**Comment by iMagic16** [comment]
how do you change to jar launcher using twitch?

also, these files @shadoxxhd - they go into /mods ?

**Comment by Flockhammer** [comment]
it's in the Minecraft options under twitch but even with that allowing it to load, I could not join any servers due to some Mod issues. I'm loading up with the fix shadoxxhd has above atm and so far so good.

**Comment by shadoxxhd** [comment]
@iMagic16 yes, instead of the original mod files.

**Comment by iMagic16** [comment]
@shadoxxhd fix confirmed working

**Comment by megadeth116** [comment]
@shadoxxhd replacing mods worked for me (didn't changed launcher)

**Comment by shadoxxhd** [comment]
@shadus please read the thread before posting. I already explained the issue, and provided a fix for it. On request I even made a video tutorial: https://youtu.be/HYN9GwFWkp4

**Comment by realral** [comment]
shadoxxhd fixed worked for me. Thank you!

**Comment by Maveriknight** [comment]
a slightly more detailed explanation of how to use those files to fix it would be appreciated for those of us somewhat rusty with doing such things.

**Comment by shadoxxhd** [comment]
@Maveriknight https://youtu.be/HYN9GwFWkp4

**Comment by shadoxxhd** [comment]
@Maveriknight the video wasn't enough?

**Comment by Maveriknight** [comment]
video got posted while I was reading stuff, it was plenty.

**Comment by Mauvai** [comment]
having removed gendustry i got a crash on thermal expansion - has anyone else experienced that?

**Comment by realral** [comment]
Yes, get the files from shadoxxhd link above and that will fix it

**Comment by Mauvai** [comment]
Forgive me if im being stupid but i only see files for gendusrty, spice of life and extra utils - will that somehow fix an crash thats reporting thermal expansion?

**Comment by shadoxxhd** [comment]
@Mauvai your crash has probably nothing to do with the language issue this thread is about. I recommend opening a new issue and posting your crash log there.

**Comment by Mauvai** [comment]
@shadoxxhd as far as i am aware it is the same bug. I got a crash on launch with gendustry first, across 3 different versions (2.2 -2.4)  and when i removed the mod it came up with thermal expansion where most people had extra utilities

**Comment by shadoxxhd** [comment]
The thermal expansion crash should be different from the gendustry one - if it wasn't, it would be fixed already.
